src/app/components/WorkDetail/DefinitionList.jsx

```diff
@@ -21,28 +22,15 @@ export const labels = {
  *
  * @param {object} props
  */
-export const DefinitionList = ({
-  dispatch, eReaderUrl, data, context,
-}) => {
+export const DefinitionList = ({ eReaderUrl, data }) => {
   data.sort((a, b) => sorting.indexOf(a[0]) - sorting.indexOf(b[0]));
-  /**
-   * onClick handler for browse searches.
-   *
-   * @param {object} event
-   * @param {string} query
-   * @param {string} field
-   */
-  const newSearchRequest = (event, query, field) => {
-    event.preventDefault();
-
-    dispatch(userQuery(query));
-    dispatch(selectedField(field));
-    dispatch(searchPost(query, field))
-      .then(() => {
-        context.router.push(`/search?q=${query}&field=${field}`);
-      });
-  };
-
+  const getIdentifier = author => (author.viaf && 'viaf') || (author.lcnaf && 'lcnaf') || 'name';
```

Comment:
What is the `&& 'viaf'` check and the `&& 'lcnaf'` check for? Won't it always be true since non-empty strings always return true?

Reply:
Ah nevermind. I see what you are doing. Not necessarily the easier to read but nice.

Reply:
I'd prefer to use ternary, but it's not allowed in eslint rules.